recipes/r-polca/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = '1.4.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-polca
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/poLCA_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/poLCA/poLCA_{{ version }}.tar.gz
+  sha256: 2e69975b5e7da8c36641bfa9453afdb4861523866b8799bec1d4eace9ab5762e
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-mass
+    - r-scatterplot3d
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-mass
+    - r-scatterplot3d
+
+test:
+  commands:
+    - $R -e "library('poLCA')"           # [not win]
+    - "\"%R%\" -e \"library('poLCA')\""  # [win]
+
+about:
+  home: http://dlinzer.github.com/poLCA
+  license: GPL-2.0-or-later
+  summary: Latent class analysis and latent class regression models for polytomous outcome variables.  Also
+    known as latent structure analysis.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
+    - dlinzer

```

Comment:
@Zyg-D who is dlinzer?

Reply:
Does he agree to be conda-forge package maintainer of the package?

Reply:
He is the author of the original package. I thought I should reference his name at least. People sometimes do it in their PRs, but myself I haven't contacted him. In this case, should I only leave `conda-forge/r`?

Reply:
Yes, you can just only add conda-forge/r or also additionally add yourself in case you want to maintain the recipe.

Reply:
@dbast - Thank you. I've done what's required.